composer.json

```diff
@@ -10,19 +10,19 @@
         }
     ],
     "require": {
-        "php": ">=5.6.0",
-        "symfony/framework-bundle": "~3.0|~4.0|~5.0",
-        "symfony/form": "~3.0|~4.0|~5.0",
-        "symfony/console": "~3.0|~4.0|~5.0",
-        "symfony/mime": "~4.0|~5.0",
-        "pimple/pimple": "~3.0",
-        "monolog/monolog": "~1.4|~2.0",
+        "php": "^7.1|8.0.*|8.1.*",
+        "symfony/framework-bundle": "^3.0|^4.0|^5.0|^6.0",
+        "symfony/form": "^3.0|^4.0|^5.0|^6.0",
+        "symfony/console": "^3.0|^4.0|^5.0|^6.0",
+        "symfony/mime": "^4.0|^5.0|^6.0",
+        "pimple/pimple": "^3.0",
+        "monolog/monolog": "^1.4|^2.0",
         "league/flysystem": "^1.1.8",
         "league/flysystem-aws-s3-v3": "^1.0.24",
         "league/flysystem-cached-adapter": "^1.0.7"
     },
     "require-dev": {
-        "phpunit/phpunit": "~5.7"
+        "phpunit/phpunit": "^5.7"
```

Comment:
not compatible with php 8 and seems like tests also not running on php 7.4 or any ci configured.